config/ngrok.example.yml

```diff
@@ -0,0 +1,17 @@
+# To find your authtoken run `cat ~/.ngrok2/ngrok.yml | grep authtoken`
+authtoken: $REPLACE
```

Comment:
NAB: I think a more familiar syntax would be <AUTH_TOKEN> and same with other replace instances in this file